sdks/java/io/mongodb/src/test/java/org/apache/beam/sdk/io/mongodb/MongoDBGridFSIOTest.java

```diff
@@ -169,7 +168,7 @@ public void testFullRead() throws Exception {
     TestPipeline pipeline = TestPipeline.create();
 
     PCollection<String> output = pipeline.apply(
-        MongoDbGridFSIO.read()
+        MongoDbGridFSIO.<String>read()
```

Comment:
minor: this generates a warning in eclipse as the `<read>` is not needed.  The other two tests have a similar issue.